arroyo-api/src/sources.rs

```diff
@@ -353,9 +368,17 @@ impl TryFrom<SchemaField> for SourceField {
             }
         };
 
+        let sql_name = match value.typ {
+            SchemaFieldType::Primitive(p) => primitive_to_sql(&p),
+            _ => ""
```

Comment:
I wasn't so sure about this case, thoughts?

Reply:
We already don't show the current type, so I think this is fine.